src/Symfony/Component/String/AbstractString.php

```diff
@@ -692,6 +692,11 @@ public function wordwrap(int $width = 75, string $break = "\n", bool $cut = fals
         return $str;
     }
 
+    /**
+     * @return static
+     */
+    abstract public function reverse(): self;
```

Comment:
should be added in alpha order as I think that's how other methods are listed

Reply:
Done.